### ISSUE TITLE
"Three Levels with master, Satellites, and Clients" chapter is not clear about client config

### ISSUE THREAD

**Issue by rmdolh** [Icinga/icinga2#5157]
Hi,

In 6.10.3, "Three Levels with master, Satellites, and Clients"

After the "Note: The master nodes do not need to know about the indirectly connected clients. Since we want to use command endpoint check configuration, we must configure the client endpoint and zone objects. In order to minimize the effort, we'll sync the client zone and endpoint config to the satellites where the connection information is needed as well."

there's a configuration of the client on master1 : 
```
[root@icinga2-master1.localdomain /]# mkdir -p /etc/icinga2/zones.d/{master,satellite,global-templates}
...
}
```
But this configuration should be located on the satellite, not on the master. So the linux prompt should be changed from master1 to satellite1 :
```
[root@icinga2-satellite1.localdomain /]# mkdir -p /etc/icinga2/zones.d/{master,satellite,global-templates}
[root@icinga2-satellite1.localdomain /]# cd /etc/icinga2/zones.d/satellite

[root@icinga2-satellite1.localdomain /etc/icinga2/zones.d/satellite]# vim icinga2-client1.localdomain.conf

object Endpoint "icinga2-client1.localdomain" {
  host = "192.168.56.111" //the satellite actively tries to connect to the client
}

object Zone "icinga2-client1.localdomain" {
  endpoints = [ "icinga2-client1.localdomain" ]

  parent = "satellite"
}

[root@icinga2-satellite1.localdomain /etc/icinga2/zones.d/satellite]# vim icinga2-client2.localdomain.conf

object Endpoint "icinga2-client2.localdomain" {
  host = "192.168.56.112" //the satellite actively tries to connect to the client
}

object Zone "icinga2-client2.localdomain" {
  endpoints = [ "icinga2-client2.localdomain" ]

  parent = "satellite"
}
```

**Comment by dnsmichi** [comment]
The configuration is intentionally managed on the master node. The satellite must not have its own zones.d, the configuration is synced from the parent zone. Did you try it out?

**Comment by rmdolh** [comment]
Yes, I tried and it didn't work till I created this on the satellite. Certainly a problem on my side.

For me, the documentation is not clear : 
* "Note: The master nodes do not need to know about the indirectly connected clients. "
* And just after, creation of the client configuration on the master : vim icinga2-client1.localdomain.conf

and a few steps after : 
* "Now it is time to define the two client hosts on the master"
* vim icinga2-client1.localdomain.conf : change the file created above ???
